On RAP, pwd from the host system will refer to the dynamic linker there,
causing symbol error from incompatibility of glibc.  Always call pwd within
RAP.

  Benda Xu <heroxbd@gentoo.org> Tue May 12 11:00:17 JST 2015

--- perl-5.16.3/dist/PathTools/Cwd.pm
+++ perl-5.16.3/dist/PathTools/Cwd.pm
@@ -331,8 +331,8 @@ $METHOD_MAP{NT} = $METHOD_MAP{MSWin32};
 # are safe.  This prevents _backtick_pwd() consulting $ENV{PATH}
 # so everything works under taint mode.
 my $pwd_cmd;
-foreach my $try ('/bin/pwd',
-		 '/usr/bin/pwd',
+foreach my $try ('@GENTOO_PORTAGE_EPREFIX@/bin/pwd',
+		 '@GENTOO_PORTAGE_EPREFIX@/usr/bin/pwd',
 		 '/QOpenSys/bin/pwd', # OS/400 PASE.
 		) {
 
